addons/l10n_th/views/report_invoice.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_invoice_document" inherit_id="account.report_invoice_document">
+        <xpath expr="//span[@t-field='o.partner_id.vat']" position="after">
+            <t t-if="o.company_id.account_fiscal_country_id.code == 'TH'" >
+                <span t-field="o.l10n_th_company_info" />
+                <span t-field="o.l10n_th_branch_number" />
+            </t>
+        </xpath>
+        <xpath expr="//h2" position="before">
+            <t t-set="use_custom_l10n_header" t-value="o.l10n_th_custom_header and o.company_id.account_fiscal_country_id.code == 'TH'"/>
+            <h2 t-if="use_custom_l10n_header">
+                <span t-if="o.debit_origin_id">Debit Note</span>
+                <span t-if="not o.debit_origin_id">Tax Invoice</span>
+                <span t-if="o.name != '/'" t-field="o.name"/>
+            </h2>
+        </xpath>
+        <xpath expr="//h2[2]" position="attributes">
```

Comment:
Inheriting with those attributes is cumbersome if multiple modules want to do the same. It is also not the first time that people want that descriptive name changed.

Reply:
what would be the best approach in this case?